source/Nevermore/IRelationalStoreConfiguration.cs

```diff
@@ -65,5 +65,10 @@ public interface IRelationalStoreConfiguration
         /// resulting in duplicate query plans being created.
         /// </summary>
         public bool DetectQueryPlanThrashing { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether to support a larger number of related documents (currently Nevermore errors at 1000) by using table valued parameters
+        /// </summary>
+        public bool SupportLargeNumberOfRelatedDocuments { get; set; }
```

Comment:
Feature switch

Reply:
What's the re removing this flag?  Delete in a few weeks/months if there are no problems in Octofront?

Reply:
Yes - once we've got this enabled in Octofront and Server for a few weeks, then we can remove this